routers/repo/pull.go

```diff
@@ -73,7 +75,19 @@ func getForkRepository(ctx *context.Context) *models.Repository {
 		ctx.Handle(500, "GetOrganizations", err)
 		return nil
 	}
-	ctx.Data["Orgs"] = ctx.User.Orgs
+	var orgs []*models.User
+	for _, org := range ctx.User.Orgs {
```

Comment:
Same as above (https://github.com/go-gitea/gitea/pull/2699/files#diff-d4e2553a87eb8b329efd7335f028903cR648) as Orgs includes also organizations where you are in team that have only read rights.